runsc/sandbox/network.go

```diff
@@ -145,6 +145,19 @@ func createInterfacesAndRoutesFromNS(conn *urpc.Client, nsPath string, hardwareG
 			continue
 		}
 
+		// Collect data from the ARP table
```

Comment:
Micro-nit for this and below comments: we try to do full sentences, so capitalization + ending in period.

Reply:
Thanks, fixed